apps/dino-tracking/backend/src/main.py

```diff
@@ -0,0 +1,174 @@
+    ReferenceFromSelectionNode,
+)
+from pathlib import Path
+from constants.yml_constants_loader import YamlFilesLoader
+from core.annotations.FE_annotations_control_services.annotation_mode_service import (
+    AnnotationModeService,
+)
+from core.object_selection.FE_prompt_services.clear_selection_sevice import (
+    ClearSelectionService,
+)
+from core.annotations.FE_annotations_control_services.outlines_trigger_service import (
+    OutlinesTriggerService,
+)
+from core.detections_tracking.threshold_service import ThresholdService
+
+from dotenv import load_dotenv
+
+from core.detections_tracking.heatmap_to_detections_node import (
+    HeatmapToDetectionsNode,
+)
+from core.detections_tracking.tracker import Tracker
+
+import depthai as dai
+
+from depthai_nodes.node import ParsingNeuralNetwork
+
+from core.annotations.outlines_overlay_node import OutlinesOverlayNode
+from core.object_selection.FE_prompt_services.click_prompt_service import (
+    ClickPromptService,
+)
+from core.annotations.dino_annotation_node import DinoAnnotationNode
+from core.encoder import Encoder
+from core.state_service import StateService
+from core.dino_similarity.similarity_heatmap_node import SimilarityHeatmapNode
+from core.object_selection.selection_mask_node import SelectionMaskNode
+
+load_dotenv(override=True)
+
+
+def main():
+    constants = YamlFilesLoader(Path(__file__).parent / "constants")
+    constants.load_all()
+
+    visualizer = dai.RemoteConnection(httpPort=8082)
+
+    device = dai.Device()
+    platform = device.getPlatformAsString()
+    print(f"Platform: {platform}")
+
+    with dai.Pipeline(device) as pipeline:
+        print("Creating pipeline...")
+
+        camera = pipeline.create(dai.node.Camera).build()
+
+        rgb_sensor = camera.requestOutput(
+            size=constants.camera.resolution,
+            type=dai.ImgFrame.Type.BGR888i,
+            fps=constants.camera.fps,
+        )
+
+        fastsam_rgb = camera.requestOutput(
+            size=constants.nn.segmentation.input_size,
+            type=dai.ImgFrame.Type.BGR888i,
+            fps=constants.camera.fps,
+        )
+
+        dino_rgb = camera.requestOutput(
+            size=constants.nn.dino.input_size,
+            type=dai.ImgFrame.Type.BGR888i,
+            fps=constants.camera.fps,
+        )
+
+        fastsam_nn = pipeline.create(ParsingNeuralNetwork).build(
+            input=fastsam_rgb, nn_source=constants.nn.segmentation.model_name
+        )
+        segmentation_out = fastsam_nn.out
+
+        dino_nn = pipeline.create(dai.node.NeuralNetwork).build(
+            input=dino_rgb,
+            nnArchive=dai.NNArchive(
+                dai.getModelFromZoo(
+                    dai.NNModelDescription(
+                        model=constants.nn.dino.model_name, platform=platform
+                    )
+                )
+            ),
+        )
+        dino_out = dino_nn.out
```

Comment:
this is not needed, correct name would be `dino_nn_out` and that is the same as `dino_nn.out` and is therefore not needed and wanted